dev/Tiles/metatile.h

```diff
@@ -0,0 +1,60 @@
+#include <stdbool.h>
+#include <stdlib.h>
+#include "../../lib/SMSlib.h"
+#include "tile.h"
+
+typedef struct MetaTile
+{
+    struct Tile tiles[16];
+    unsigned char numTiles;
+    unsigned char numTilesX;
+    unsigned char numTilesY;
+} MetaTile;
+
+MetaTile* MetaTile_Create
+(
+    Tile tiles[16],
```

Comment:
There is no point in specifying the size of this array, since it will decay to a pointer anyway.

```suggestion
    Tile tiles[],
```

Reply:
I'll double check this one but I think I recall getting a compiler error when I didn't add the length.